pkg/boot/params.go

```diff
@@ -118,13 +122,26 @@ type Params struct {
 	LocalPK          cipher.PubKey `json:"local_pk"` // Not actually encoded to bps.
 	LocalSK          cipher.SecKey `json:"local_sk"`
 
+	// only on dmsghttp
+	DMSGHTTP string `json:"dmsghttp,omitempty"`
```

Comment:
Instead of string we can have this of type  [`dmsgHTTPServersData`](https://github.com/ersonp/skywire/blob/d18280346f59a769fd008057da76983896b5daaf/cmd/skywire-cli/commands/config/gen.go#L195)

Reply:
Actually because of use this data in boot parameter and write on skybian image,  use string.